supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,299 @@
+[[red-hat-single-sign-on-conventions]]
+
+[discrete]
+[[access-token]]
+==== image:images/yes.png[yes] access token
+An access token is a token that can be provided as part of an HTTP request that grants access to the service being invoked on. This is part of the OpenID Connect and OAuth 2.0 specification.
```

Comment:
All of these descriptions are missing the `*Description*:` labels

Reply:
Uff! apologies. On it